source/administration/object-management/data-compression.rst

```diff
@@ -0,0 +1,191 @@
+
+By default, MinIO compresses the following extensions and types:
+
+* File extensions: ``.txt``, ``.log``, ``.csv``, ``.json``, ``.tar``, ``.xml``, ``.bin``
+* Media types: ``text/*``, ``application/json``, ``application/xml``, ``binary/octet-stream``
+
+.. admonition:: Default excluded extensions and types are never compressed
+   :class: note
+
+   Some objects cannot be efficiently compressed.
+   MinIO will not attempt to compress these objects, even if they are specified in :mc-conf:`~compression.extensions` or :mc-conf:`~compression.mime_types` arguments.
+
+The sections below describe how to configure compression for the desired file extensions and media types.
+
+Compress All Compressible Objects
++++++++++++++++++++++++++++++++++
+
+To compress all objects except the default excluded ones, use :mc-cmd:`mc admin config set` to set the :mc-conf:`compression` key :mc-conf:`~compression.extensions` and :mc-conf:`~compression.mime_types` options to empty lists.
+
+The following compresses all objects, except the default incompressible objects:
+
+.. code-block:: shell
+   :class: copyable
+
+   mc admin config set ALIAS compression extensions= mime_types=
+
+- Replace ``ALIAS`` with the :mc:`alias <mc alias>` of a configured MinIO deployment.
+
+  
+Compress Objects by File Extension
+++++++++++++++++++++++++++++++++++
+
+To compress objects with certain file extensions, use :mc-cmd:`mc admin config set` to set the :mc-conf:`compression` key :mc-conf:`~compression.extensions` option to a list of the desired file extensions.
+
+The following command compresses files with the extensions ``.bin`` and ``.txt``:
+
+.. code-block:: shell
+   :class: copyable
+
+   mc admin config set ALIAS compression extensions=".bin, .txt"
+
+- Replace ``ALIAS`` with the :mc:`alias <mc alias>` of a configured MinIO deployment.
+
+The new list of file extensions replaces the previous list.
+To add or remove an extension, repeat the :mc-conf:`~compression.extensions` command with the complete list of extensions to compress.
+
+The following command adds ``.pdf`` to the list of file extensions from the previous example:
+
+.. code-block:: shell
+   :class: copyable
+
+   mc admin config set ALIAS compression extensions=".bin, .txt, .pdf"
+
+- Replace ``ALIAS`` with the :mc:`alias <mc alias>` of a configured MinIO deployment.
+
+
+Compress Objects by Media Type
+++++++++++++++++++++++++++++++
+
+To compress objects of certain types, use :mc-cmd:`mc admin config set` to set the :mc-conf:`compression` key :mc-conf:`~compression.mime_types` option to a list of the desired types.
+
+The following example compresses files of types ``application/json`` and ``image/png``:
+
+.. code-block:: shell
+   :class: copyable
+
+   mc admin config set ALIAS compression mime_types="application/json, image/png"
```

Comment:
since we ignore `*.png` maybe a different image format would be better here. TIFF? Those are uncompressed by default. or RAW

Reply:
though RAW apparently has different mime types depending, so BMP maybe